aspnetcore/fundamentals/routing.md

```diff
@@ -359,6 +363,20 @@ Refer to [.NET Framework Regular Expressions](https://docs.microsoft.com/dotnet/
 
 To constrain a parameter to a known set of possible values, use a regular expression. For example, `{action:regex(^(list|get|create)$)}` only matches the `action` route value to `list`, `get`, or `create`. If passed into the constraints dictionary, the string `^(list|get|create)$` is equivalent. Constraints that are passed in the constraints dictionary (not inline within a template) that don't match one of the known constraints are also treated as regular expressions.
 
+## Parameter transformer reference
+
+Parameter transformers execute when generating a link for a `Route`. Parameter transformers take the parameter's route value and transform it to a new string value. The transformed value will be used in the generated link. For example, a custom slugify parameter transformer in route pattern `blog\{article:slugify}` with `Url.Action(new { article = "MyTestArticle" })` will generate `blog\my-test-article`. A parameter transformer is configured using <xref:Microsoft.AspNetCore.Routing.RouteOptions.ConstraintMap>.
+
+Parameter transformers are also used by frameworks to transform the URI that an endpoint will resolve to. For example, ASP.NET Core MVC will use parameter transformers to transform the route value used to match an `area`, `controller`, `action` and `page`.
```

Comment:
Should we show an example of how to configure this pages?

Reply:
Pardon? I added info about the MVC API convention. It plus convention routes are the two main ways to use this.

Reply:
I think I failed to see that before, or it wasn't updated yet? The info about RouteTokenTransformerConvention is there now.

Reply:
I added in response to your first comment. I wasn't sure if it was what you wanted